v2/src/pages/About/index.js

```diff
@@ -118,10 +118,9 @@ const features = [
 
 export default function About() {
   return (
-    <DefaultLayout className="bg-gray-50" title="About Us">
-
+    <DefaultLayout  title="About Us">
```

Comment:
There's a library called react-helmet which I think should be used to set things like titles.